ocean_lib/models/vetest/test_ve_ocean.py

```diff
@@ -19,6 +20,9 @@
 @pytest.mark.unit
 def test1(config, factory_deployer_wallet, ocean_token, veOCEAN):
     # inspiration from df-py/util/test/veOcean/test_lock.py
+    brownie.network.gas_limit(chain.block_gas_limit)
+    priority_fee(chain.priority_fee)
```

Comment:
There are two references to `chain` but I don't see a line like `from brownie.network import chain`. Is it set in a higher-up conftest? Or simply an omission here?

Reply:
ah yes, I see it now. thanks